### PR TITLE
Fix a couple bugs in the import script

### PR DESCRIPTION
**Pull request by gbleaney**
Thanks for putting it together. I ran into a few problems when running it, so I figured I'd send you a PR to update them

### CODE REVIEW

File: ImportStats v1.js [hunk 1]
```diff
@@ -182,18 +182,23 @@
     
     jf.parseStatblock = function(statblock) {
 
+        log("Dirty block:\n " + statblock);
```
Comment (gbleaney) [inline ImportStats v1.js:185]: Just dumped in a bunch of extra logging to help debugging future problems

File: ImportStats v1.js [hunk 2]
```diff
@@ -182,18 +182,23 @@
...
         processSection(section);
         return section.attr.name;
     }
 
     function clean(statblock) {
         statblock = unescape(statblock);
+        log("Unescaped block:\n " + statblock);
         statblock = statblock.replace(/–/g, '-');
-        statblock = statblock.replace(/<br[^>]*>/g, '#').replace(/(<([^>]+)>)/ig, "");
+        statblock = statblock.replace(/<p>/g, '#').replace(/(<([^>]+)>)/ig, "");
```
Comment (gbleaney) [inline ImportStats v1.js:201]: This changes from targeting `<br>` tags to `<p>` tags. My observation is that roll20 *always* puts in the `<p>` tags, but doesn't always put in the `<br>`s. This fixes the problem discussed [here](https://app.roll20.net/forum/post/10800206/script-importstats-take-monster-stat-blocks-and-import-them-into-roll20-5e-npc-sheets).

File: ImportStats v1.js [hunk 3]
```diff
@@ -239,7 +244,7 @@
         }
 
         // Power
-        regex = /(?:#|\.\s+)([A-Z][\w-]+(?:\s(?:[A-Z][\w-]+|[\(\)\d/-]|of)+)*)(?=\s*\.)/g;
+        regex = /(?:#\s*)([A-Z]['\w-]+(?:\s(?:[A-Za-z]['\w-]+|\([\w\d/]+\)|-|of)+)*)(?=\s*\.)/g;
```
Comment (gbleaney) [inline ImportStats v1.js]: This fixes a few problems:
1. It allows (1/day) type traits
2. It allows traits that have lower case words like "and" or "with"
3. It allows traits with `'` in them

File: README.md [hunk 4]
```diff
@@ -32,9 +32,10 @@ Use Ctrl+a and Ctrl+c to select all of that document and copy it, then paste it
 
 ![script save](https://user-images.githubusercontent.com/103297938/162561605-54b977d8-091f-42f9-b500-0ca2ef56e35e.png)
 
-Once you save you are ready to setup a Macro in your roll20 campaign, log into your campaign and select Macros in the upper right, you might never have used this tab before now: 
+Once you save you are ready to setup a Macro in your roll20 campaign, log into your campaign and select Collection in the upper right, you might never have used this tab before now:
```
Comment (gbleaney) [inline README.md:35]: UI was updated :)

File: README.md [hunk 5]
```diff
@@ -65,8 +66,6 @@ Here are some of my findings to help with the parsing:
 	-Sometimes copying the end of a line to the beginning of the next and pasting that helps.
 	-Making all of an ability’s text a single line is often important if you are having issues.
 	-Traits often isn’t in the text - it can help to add a new line with “Traits” directly between the stat block section and the start of your Traits.
-	-Apostrophes will mess up Ability names, so leave those out. Devil’s Sight messed me up a lot.
```
Comment (gbleaney) [inline README.md:68]: Deleted because I fixed them

**Review by Zanthox — APPROVED** [review]

File: ImportStats v1.js (hunk 2 above)
Comment (Zanthox) [reply ImportStats v1.js:201]: Thanks! Can't wait to try it.

**Review by Zanthox — APPROVED** [review]

**Review by Zanthox — COMMENTED** [review]
Good!

**Comment by Zanthox** [comment]
Hey Graham, I've had a couple issues as well as a couple users message me that they cant use the script after your update. I've reverted to my old code and when I sent it to them they were able to successfully import. 
The issue was no actions were getting made and all the statblock would get dumped into a single trait. 
If you dont wanna review the changes you made I can just roll it back.

**Comment by gbleaney** [comment]
@Zanthox Feel free to revert for now to unblock people.

Can you get me some examples of stat blocks that were failing to import though? I'll see if I can make my changes compatible with them.

**Comment by Zanthox** [comment]
They both used the given grandfather of assassins example, here is one of their results: 
image

**Pull request merged**